src/jitlayers.cpp

```diff
@@ -1307,7 +1309,7 @@ void jl_merge_module(Module *dest, std::unique_ptr<Module> src)
 
 // optimize memory by turning long strings into memoized copies, instead of
 // making a copy per object file of output.
-void jl_jit_share_data(Module &M)
+void JuliaOJIT::shareStrings(Module &M)
```

Comment:
Just as a note, we discussed with Lang that this would be something we could do at the JITLink level instead, but that will have to wait until all platforms use JITLink.